app/controllers/password_reset_controller.rb

```diff
@@ -0,0 +1,31 @@
+class PasswordResetController < ApplicationController
+  def enter_email
+  end
+  
+  def confirm_password_reset
+    user = User.find_by_email_address params[:email]
```

Comment:
`find_by_*` methods are not recommended anymore as they are less performant than `find_by`. You can use `find_by(email_address: params[:email])`.